### ISSUE TITLE
HOW TO RECORD AN ISSUE

### ISSUE THREAD

**Issue by theoo** [theoo/resman#1]
In order to help bugfixes, you can copy-paste the following template. All fields are mandatory.

First step: ensure your new issue is unique, if related to another issue please provide the link in the description.

**What, when and where is the expected behaviour?**
_TLDR; Provide Time, Space and expectation information._
Example: When I click the button called "lottery" on the top left of the screen in the System/Settings/Wealth submenu, I would like to become rich and famous.
This should look like this: $$$ :+1: 
Keep in mind that I have to reproduce the bug.

**What went wrong?**
_TLDR; Provide information in contrast with the expectation_
Well, I'm famous but still not rich. My shrink says the problem comes from this button, which obviously doesn't work. When I click nothing happen, really - nothing.
This look like this instead: :+1: :-(

**Screenshot**
_TLDR; Well, a picture says a thousand words_
![image](https://cloud.githubusercontent.com/assets/390145/22264985/6b3c01a8-e27a-11e6-9931-1c94956241db.png)
